mne/viz/topomap.py

```diff
@@ -2382,3 +2382,86 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
     fig.canvas.draw()
     plt_show(show)
     return fig
+
+
+def _trigradient(x, y, z):
+    """Take gradients of z on a mesh."""
+    from matplotlib.tri import CubicTriInterpolator, Triangulation
+    tri = Triangulation(x, y)
+    tci = CubicTriInterpolator(tri, z)
+    dx, dy = tci.gradient(tri.x, tri.y)
+    return dx, dy
+
+
+def plot_arrowmap(data, info_from, info_to=None, scale=1e-10, axes=None):
+    """Plot arrow map.
+
+    Parameters
+    ----------
+    data : array, shape (n_channels,)
+        The data values to plot.
+    info_from : instance of Info
+        The measurement info from data to interpolate from.
+    info_to : instance of Info | None
+        The measurement info to interpolate to.
+    scale : float, default 1e-10
+        To scale the arrows
+    axes : instance of Axes | None
+        The axes to plot to. If None, the current axes will be used.
```

Comment:
This is not true, a new figure will be created

Reply:
fixed